metadata_backend/api/operators.py

```diff
@@ -443,9 +442,9 @@ def _generate_accession_id(self) -> str:
 
         Will be replaced later with external id generator.
         """
-        sequence = "".join(secrets.choice(string.digits) for i in range(16))
+        sequence = "".join(str(uuid.uuid4()))
```

Comment:
Generating UUID here will result in the standard format that would look for example like this:
`123e4567-e89b-12d3-a456-777888999000`

If we don't want the hyphens, you can get a version if the UUID without the hyphens by replacing the line with this:
`sequence = "".join(uuid.uuid4().hex)`

Reply:
even simpler is `uuid4().hex` it generates string by default https://docs.python.org/3/library/uuid.html#uuid.UUID.hex

Reply:
and yes please use `.hex`